cypress/e2e/bookStore.cy.js

```diff
@@ -2,10 +2,27 @@
 
 describe('Book Store app', () => {
   before(() => {
-
+    cy.visit('https://demoqa.com/books');
```

Comment:
Why do you start with `/books` page just to switch to login right away?

Reply:
I start with "/books" because it is not possible to log in on the previous page, especially since the task concerns actions on books